src/west/manifest.py

```diff
@@ -303,7 +303,7 @@ def validate(data):
         pykwalify.core.Core(source_data=data,
                             schema_files=[_SCHEMA_PATH]).validate()
     except pykwalify.errors.SchemaError as se:
-        raise MalformedManifest(se._msg) from se
+        raise MalformedManifest(se.msg) from se
```

Comment:
Did you intent to mean:
```suggestion
        raise MalformedManifest(se.msg()) from se
```

Reply:
At least on my system's pykwalify, msg is a property and therefore should be accessed like an attribute, so this wouldn't work.

Do you have a specific pykwalify version where this is required?

Reply:
The reason I ask, is because I see both a method:
https://github.com/Grokzen/pykwalify/blob/master/pykwalify/errors.py#L94

and a property:
https://github.com/Grokzen/pykwalify/blob/master/pykwalify/errors.py#L55

but in your commit message you write:
> The SchemaError type's error message should be accessed via the msg()
property,

so bacause you used `msg()` I simply got unsure whether you meant the property or the method `msg()`.
that was the only reason for asking.

Reply:
Thanks for clarifying.
Approved.

Reply:
> The reason I ask, is because I see both a method:
> https://github.com/Grokzen/pykwalify/blob/master/pykwalify/errors.py#L94
> 
> and a property:
> https://github.com/Grokzen/pykwalify/blob/master/pykwalify/errors.py#L55

The code that starts on line 94 is written in a tricky style that's IMO a bit hard to read, but it's a property, not a method. I'm going to copy/paste the entire thing here:

```
def msg():
    doc = """ """

    def fget(self):
        return self._msg

    def fset(self, value):
        assert isinstance(value, basestring), "argument is not string"
        self._msg = value

    return locals()
msg = property(**msg())
```

Notice the final `msg = property(**msg())`. The first definition of `msg`, using `def`, is overwritten immediately by this statement.

So the final value of `msg` is the return value of `property()`, whose arguments are taken by passing the return value of `msg()` as kwargs.

In turn, the `msg()` call returns the dictionary `locals()`, which looks like this:

```
>>> msg()
{'doc': ' ',
 'fget': <function __main__.msg.<locals>.fget(self)>,
 'fset': <function __main__.msg.<locals>.fset(self, value)>}
```

So it's as if you defined

```
msg = property(doc=' ', fget=<some function>, fset=<another function>)
```

and its final value is a "real" property with a getter and a setter function. That's why errors.py on line 55 can treat it like a property; it is one.

Reply:
I know.
The thing I tried to say, was that because your commit message said `msg()` but you used the property `msg` I simply was unsure what your really intented to do.

Reply:
Ah, OK, I understand your meaning now, thanks.